src/SFA.DAS.EmployerCommitmentsV2.Web/Views/Apprentice/WhyStopApprenticeship.cshtml

```diff
@@ -0,0 +1,69 @@
+﻿@inject ILinkGenerator LinkGenerator
+@using SFA.DAS.CommitmentsV2.Shared.Extensions
+@using SFA.DAS.CommitmentsV2.Types
+@using SFA.DAS.EmployerCommitmentsV2.Web.Extensions
+@using SFA.DAS.EmployerCommitmentsV2.Web.Models.Apprentice
+@model SFA.DAS.EmployerCommitmentsV2.Web.Models.Apprentice.WhyStopApprenticeshipViewModel
+
+@{
+    ViewBag.Title = "Why do you want to stop this apprenticeship?";
+    ViewBag.Section = "apprentices";
+    ViewBag.PageId = "why-stop-apprenticeship";
+    ViewBag.GaData.Vpv = "/accounts/apprentices/manage/why-stop-apprenticeship";
+}
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <partial name="_ValidationSummary" />
+        <h1 class="govuk-heading-xl">@ViewBag.Title</h1>   
+        
+        <form method="post">
+            @Html.AntiForgeryToken()
+
+            <input type="hidden" asp-for="CurrentStatus" />
+            <div class="govuk-form-group  @(ViewData.ModelState.IsValid ? " " : " govuk-form-group--error")">
+                <fieldset class="govuk-fieldset">
```

Comment:
Every `<fieldset>` needs a `<legend>` as the first element to describe the contents and give context to the form fields contained. It doesn't have to be visible. 
I suggest adding the following `<legend class="govuk-visually-hidden">Select an option why you want to stop this apprenticeship</legend>`